Source/UI/NewUI/BillingForm/Default Implementation/Elements/Country/Country.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+class DefaultCountryFormButtonStyle: ElementButtonStyle {
+    var image: UIImage? = "arrow_blue_right".vectorPDFImage(forClass: CheckoutTheme.self)
```

Comment:
Shouldn't this be imported as a standard UIImage? (ie: `UIImage(named: ...`)

Reply:
same 
https://github.com/checkout/frames-ios/pull/158#discussion_r882677314